source/gui/settingsDialogs.py

```diff
@@ -1623,6 +1623,30 @@ def makeSettings(self, settingsSizer):
 		wordWrapText = _("Avoid splitting &words when possible")
 		self.wordWrapCheckBox = sHelper.addItem(wx.CheckBox(self, label=wordWrapText))
 		self.wordWrapCheckBox.Value = config.conf["braille"]["wordWrap"]
+		# Translators: The label for a setting in braille settings to select how the context for the focus object should be presented on a braille display.
+		focusContextPresentationLabelText = _("Focus context presentation:")
+		self.focusContextPresentations=[
+			("fill",
+			# Translators: The label for a braille focus context presentation setting that
+			# shows as much as possible focus context information if the focus object doesn't fill up the whole display.
+			# This was the pre NVDA 2017.3 default.
+			_("Always fill display")),
+			("scroll",
+			# Translators: The label for a braille focus presentation setting that
+			# always shows the object with focus at the very left of the braille display (i.e. you will have to scroll back for focus context information).
+			_("Only when scrolling back")),
+			("hybrid",
+			# Translators: The label for a braille focus presentation setting that
+			# only shows as much as possible focus context information when the context has changed.
+			_("Fill display for context changes")),
+		]
+		focusContextPresentationChoices = [x[1] for x in self.focusContextPresentations]
+		self.focusContextPresentationList = sHelper.addLabeledControl(focusContextPresentationLabelText, wx.Choice, choices=focusContextPresentationChoices)
+		try:
+			index=[x[0] for x in self.focusContextPresentations].index(config.conf["braille"]["focusContextPresentation"])

```

Comment:
Since we end up having to run this list comprehension again later, I think it'd make sense to save this on self. That would also make the code more readable in both places.